src/docfx/validation/ContentValidator.cs

```diff
@@ -76,7 +87,12 @@ public void ValidateHeadings(FilePath file, List<ContentNode> nodes, bool isIncl
         {
             if (TryGetValidationDocumentType(file, isInclude, out var documentType))
```

Comment:
`isInclude` should be removed?

Reply:
Retain this to future feature.

Reply:
for what feature? learn content validation?

Reply:
for heading validation, we can remove it at least.

Reply:
Removed. `isInclude` is still there for `ValidateCodeBlock`, need cleanup in future.